marshmallow_utils/schemas/multipoint.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+#
+# Marshmallow-Utils is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""GeoJSON MultiPoint Schema."""
+
+from geojson import MultiPoint
+from marshmallow import Schema, validates
+from marshmallow.exceptions import ValidationError
+from marshmallow.fields import Constant, Float, List
+
+
+class MultiPointSchema(Schema):
+    """GeoJSON MultiPoint schema."""
+
+    coordinates = List(List(Float), required=True)
+    type = Constant('multipoint')
+
+    @validates("coordinates")
+    def validate_data(self, data, **kwargs):
+        """Validate point.
+
+        From the `coordinates`, the first two elements are longitude
+        and latitude, or easting and northing, precisely in that order and
+        using decimal numbers. Altitude or elevation MAY be included as an
+        optional third element.
+        """
+        if not data:
+            raise ValidationError({"geojson": "coordinates must be present."})
```

Comment:
> Isn't that handled by the required=True above on the coordinates field?

Correct, it was legacy code. First I tried to have a single "validates_schema" on the `GeoJSONSchema`, and a simple attr `geojson_cls`. Because as you can see the validation function is *the same* except for the geojson object they invoke for validation. In this case, this check was needed. However, I did not manage to make it work, for some reason the validation decorator was not triggered (even though `OneOfSchema` inherits from `Schema`...). Some issues are open also about `post_load` and company not being triggered.

Proposal: Open an issue, and address this when we add more fields/schemas

> Also I think passing a dict to the ValidationError hides the real field that has the error (unless we want to establish a convention for exposing GeoJSON error under a specific key)

My idea was to actually expose GeoJSON since we are validating with it... would be nice to tell the user that in a meaningful way.